decodeSignal.m

```diff
@@ -1,4 +1,23 @@
-function [decoded, actualBER] = decodeSignal(signal)
-  actualBER = 0.03;
-  decoded = [0 1 0 1]; 
-end
\ No newline at end of file
+function [decoded,actualBER] = decodeSignal(signal)
+    signalLength = length(signal);
+    decoded = zeros(signalLength/3,1);
```

Comment:
@krzysiekissel Tutaj tworzone są wiersze, a wszystko powinno być w jednym wierszu. Jak odpalisz main.m to widać o co chodzi

Reply:
Chodzi o uproszczenie ? decoded = zeros(length(signal)/3,1)

Reply:
Oki teraz czaje, wszystko jasne